packages/braintree-integration/src/braintree-ach/braintree-ach-payment-strategy.ts

```diff
@@ -210,7 +151,62 @@ export default class BraintreeAchPaymentStrategy implements PaymentStrategy {
         return shouldVerifyVaultingInstrument ? this.tokenizePayment(payment) : null;
     }
 
-    private getBankDetails(paymentData: WithBankAccountInstrument): BankAccountSuccessPayload {
+    private async preparePaymentData(
```

Comment:
There is no changes in this methods, I just moved tokenize related method above this prepare payment data methods